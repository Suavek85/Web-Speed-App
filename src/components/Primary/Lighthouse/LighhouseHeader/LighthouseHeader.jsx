import React from 'react'

import WebOptimizeWrapper from '../../../svgs/WebOptimizeWrapper'
import styles from './LighthouseHeader.scss'
import { LIGHTHOUSE_TITLE } from '../../../../constants/content'

export default function LighthouseHeader() {

  const PERFORMANCE = 'Performance'

  return (
    <div className={ styles.blockHeader }>
      <WebOptimizeWrapper />
      <div style={{display: 'flex', flexDirection:'column', alignItems:'center'}}>
        <h1 style={{color: 'white', marginBottom: '0'}}>{ LIGHTHOUSE_TITLE }</h1>
        <p style={{color: '#24305e', fontSize: '18px', marginTop: '10px', fontWeight: '500'}} >{ PERFORMANCE }</p>
      </div>
    </div>
  )
}

