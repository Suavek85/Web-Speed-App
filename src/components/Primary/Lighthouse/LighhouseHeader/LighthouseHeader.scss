@import '../../../../scss/styles.scss';

.lighthouse__header {
  display: flex; 
  align-items: center;
  height: 120px;
  padding: 30px 40px;
  border-bottom: $ws-lighthouse-border-light;
}

.lighthouse__title-wrapper {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  padding-left: 20px;
}

.lighthouse__title {
  color: papayawhip;
  margin-bottom: 0;
}

.lighthouse__subtitle {
  color: cadetblue;
  font-size: 20px;
}


:export {
    blockHeader: lighthouse__header;
    blockTitleWrapper: lighthouse__title-wrapper;
    blockTitle: lighthouse__title;
    blockSubTitle: lighthouse__subtitle;
}