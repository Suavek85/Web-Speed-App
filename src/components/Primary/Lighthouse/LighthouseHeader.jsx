import React from 'react'

import WebOptimizeWrapper from '../../svgs/WebOptimizeWrapper'
import styles from './Lighthouse.scss'



export default function LighthouseHeader() {
  return (
    <div className={ styles.blockHeader }>
      <WebOptimizeWrapper />
      <h1>Lighthouse<br/>Score</h1>
    </div>
  )
}
