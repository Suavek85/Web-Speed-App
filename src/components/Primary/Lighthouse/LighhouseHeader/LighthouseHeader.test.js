import React from 'react'
import { shallow } from 'enzyme'
import LighthouseHeader from './LighthouseHeader'

describe('LighthouseHeader snapshot', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<LighthouseHeader />)
    expect(wrapper).toMatchSnapshot()
  })

})