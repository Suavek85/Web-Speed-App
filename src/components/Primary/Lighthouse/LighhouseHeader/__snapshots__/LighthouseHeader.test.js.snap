// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LighthouseHeader snapshot should match the snapshot 1`] = `
<div
  className="block"
>
  <WebOptimizeWrapper />
  <div>
    <h1>
      Lighthouse Score
    </h1>
    <h2>
      Overall Performance
    </h2>
  </div>
</div>
`;
